sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -355,6 +358,12 @@ private void setArchitectures(final @NotNull Device device) {
       device.setLocale(locale.toString()); // eg en_US
     }
 
+    final @NotNull List<Integer> cpuFrequencies = CpuInfoUtils.getInstance().readMaxFrequencies();
```

Comment:
Just to confirm, could this method be called in the main thread? Because it involves I/O the first time it is invoked